cluster/examples/kubernetes/rook-tools.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: rook-tools
   namespace: rook
 spec:
-  hostNetwork: false
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
should this dns policy only be set if `hostNetwork: true`?

Reply:
@travisn When `dnsPolicy: ClusterFirstWithHostNet` is set the cluster internal dns are set first and then the external "host" dns servers. It doesn't matter that much, but I can remove it if wanted.

Reply:
Ok thanks for the clarification. If you could just rebase, I'll plan on merging it.